experiments/testing_archive/test_experiment_exceptions.py

```diff
@@ -0,0 +1,151 @@
+"""
+Test module for the experiment_handler/experiment_prototype code.
+It is run simply via 'python3 test_experiment_exceptions.py' and will go through all tests
+in the experiment_tests.csv file as well as the hardcoded tests here that don't fit nicely into
+a csv file.
+
+The csv file format is:
+[#][experiment file module import name]::[regex error message]
+
+The [#] is an optional comment, and that line will be removed
+An example of a test line is:
+testing_archive.my_test_experiment.py::Regex line that * matches the ExperimentException err msg
+
+References:
+https://stackoverflow.com/questions/32899/how-do-you-generate-dynamic-parameterized-unit-tests-in-python
+https://docs.python.org/3/library/unittest.html
+
+:copyright: 2020 SuperDARN Canada
+:author: Kevin Krieger
+"""
+
+
+import unittest
+import os
+import sys
+import argparse
+
+BOREALISPATH = os.environ['BOREALISPATH']
+sys.path.append(BOREALISPATH)
+# Need to hardcode this, as unittest does weird things when you supply an argument on command line,
+# or if you use argparse. There is probably a better way
+input_test_file = "PATH_TO_/experiment_tests.csv"
```

Comment:
can refer to BOREALISPATH though right? 
```suggestion
input_test_file = BOREALISPATH + "/experiments/testing_archive/experiment_tests.csv"
```